docker/Dockerfile.gpu

```diff
@@ -1,11 +1,12 @@
-FROM nvidia/cuda:12.2.0-runtime-ubi8 as base
+FROM nvidia/cuda:11.7.1-runtime-ubi8 as base
 
 ARG cnlpt_version
 
 RUN yum -y install python39 python39-pip
-RUN pip3.9 install cython torch
 RUN pip3.9 install cnlp-transformers==$cnlpt_version
 
+# pytorch can't find the cudnn library with our setup, so just point at it directly
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/python3.9/site-packages/nvidia/cudnn/lib/"
```

Comment:
I don't know why this wasn't auto-discoverable. Maybe something to do with an oddity that changed between the 10.x and 11.x cuda images...?